src/commands/download_file.ts

```diff
@@ -19,13 +19,14 @@ new CLICommand({
 		const destFolderPath = resolvePath(parameters.getParameterValue(LocalFilePathParameter) || './');
 
 		if (await parameters.getIsPrivate()) {
-			const driveId = parameters.getRequiredParameterValue(DriveIdParameter);
-			const driveKey = await parameters.getDriveKey({ driveId: EID(driveId) });
 			const wallet = await parameters.getRequiredWallet();
 			const ardrive = cliArDriveFactory({
 				wallet,
 				feeMultiple: parameters.getOptionalBoostSetting()
 			});
+			const driveIdString = parameters.getParameterValue(DriveIdParameter);
```

Comment:
This is still required at the parameter declaration level

Reply:
Even for public downloads

Reply:
95e614a

Reply:
d86bb4f7a9784e34c2bd40092b72c0e9cc07886b

Reply:
Removed